### ISSUE TITLE
add more files to the project

### ISSUE THREAD

**Issue by jinsanko** [Xilinx/LSTM-PYNQ#2]
In order to synth with Vivado_HLS to synth the code, I have to add input_handling.cpp to hls-syn.tcl.
Is it reasonable?

Best regards
Jinsan Ko

**Comment by ussamazahid96** [comment]
Yes I did the same. and after that bitstream was synthesized successfully.

**Comment by yuginhc** [comment]
> I also modify the line 102~104 "make-hw.sh" to point to the correct folder in order to generate the bitstream.
> 
> TEST_INPUT="$LSTM_ROOT/../../tests/Test_images/$DATASET/$NETWORK/test_image.txt"
> TEST_RESULT="$LSTM_ROOT/../../tests/Test_images/$DATASET/$NETWORK/test_image_gt.txt"
> ALPHABET="$LSTM_ROOT/../datasets/$DATASET/alphabet.txt"

Can you share the worked hls-syn.tcl and make-hw.sh?

**Comment by yuginhc** [comment]
> @yuginhc
> [1.zip](https://github.com/Xilinx/LSTM-PYNQ/files/2512922/1.zip)
> Please let me know the result. Thank.

Thank you, it works now. 
But I found that the utilization is so high even with W2A2. It can not fit xc7z020clg400-1
![resource](https://user-images.githubusercontent.com/37980848/47472711-a4458880-d84a-11e8-9f18-114b92013fc4.png)

Is it same with you?

**Comment by dongzwhitsz** [comment]
I run the code  as following:

    import lstm
    hw_ocr = lstm.PynqFrakturOCR(lstm.RUNTIME_HW)

but i get the following problem.

OSError: cannot load library '/usr/local/lib/python3.6/dist-packages/lstm/libraries/fraktur/W5A5/libhw-fraktur-W5A5-ocr-pynq.so': libsds_lib.so: cannot open shared object file: No such file or directory.  Additionally, ctypes.util.find_library() did not manage to locate a library called '/usr/local/lib/python3.6/dist-packages/lstm/libraries/fraktur/W5A5/libhw-fraktur-W5A5-ocr-pynq.so

I try a lot,but I failed. Could I get some suggestion?

**Comment by Adnan86** [comment]
> @yuginhc
> [1.zip](https://github.com/Xilinx/LSTM-PYNQ/files/2512922/1.zip)
> Please let me know the result. Thank.

I followed your instructions, but still, have this error in terminal: vivado not found in path

**Comment by giuliogamba** [comment]
@jinsanko We do welcome pull request if you were able to solve a problem with the current repo organization.
